docs/astro/src/content/docs/reference/std-widgets/views/textedit.mdx

```diff
@@ -106,3 +106,7 @@ handle keys before `TextEdit` does. Return `accept` to indicate that you've hand
 Invoked when a key is released, the argument is a <Link type="KeyEvent" /> struct. Use this callback to
 handle keys before `TextEdit` does. Return `accept` to indicate that you've handled the event, or return
 `reject` to let `TextEdit` handle it.
+
+## Accessibility
```

Comment:
I don't think we need to mention that for the widgets. If yes, then it is missing in all the other widgets.

Reply:
You're right, those should just work out of the box.